src/main/java/org/motechproject/newebodac/dto/VaccineeDto.java

```diff
@@ -0,0 +1,70 @@
+package org.motechproject.newebodac.dto;
+
+import java.util.Set;
+import lombok.Getter;
+import lombok.Setter;
+import org.motechproject.newebodac.validate.Uuid;
+
+public class VaccineeDto {
+
+  @Getter
+  @Setter
+  @Uuid
+  private String id;
+
+  @Getter
+  @Setter
+  private String vaccineeId;
+
+  @Getter
+  @Setter
+  private String name;
+
+  @Getter
+  @Setter
+  private String gender;
+
+  @Getter
+  @Setter
+  private Integer age;
+
+  @Getter
+  @Setter
+  private Integer yearOfBirth;
+
+  @Getter
+  @Setter
+  private String phoneNumber;
+
+  @Getter
+  @Setter
+  private String alternatePhoneNumber;
+
+  @Getter
+  @Setter
+  private String preferredLanguageName;
+
+  @Getter
+  @Setter
+  private String preferredLanguageId;
+
+  @Getter
+  @Setter
+  private String address;
+
+  @Getter
+  @Setter
+  private String groupName;
+
+  @Getter
+  @Setter
+  private String groupId;
+
+  @Getter
+  @Setter
+  private Set<ExtraFieldDto> extraFields;
+
+  @Getter
+  @Setter
+  private Set<VisitDto> visits;
```

Comment:
I don't think this will be used on the frontend, it's better to add it when it will be needed that have the field unused